src/librbd/crypto/CryptoObjectDispatch.cc

```diff
@@ -32,7 +32,7 @@ struct C_AlignedObjectReadRequest : public Context {
     uint64_t object_no;
     io::ReadExtents* extents;
     IOContext io_context;
-    const ZTracer::Trace parent_trace;
+    const jspan_context parent_trace{false, false};
```

Comment:
why did you create the trace here, and not just passed the null trace (as in the previous examples)?